pre-commit/web_lint_check.py

```diff
@@ -22,18 +22,19 @@ def web_lint_check():
         "-C",
         DIR,
         "prettier",
-        f"PRE_ARGS=-- -c {file_paths}",
+        f"PRE_ARGS=-- --write -c {file_paths}",
     ]
 
     # TODO: replace it with `match` if we support python v3.10
     if target == "js":
-        run_command += ["eslint", f"ES_ARGS={file_paths}"]
+        run_command += ["eslint", f"ES_ARGS=-- --fix {file_paths}"]
     elif target == "css":
-        run_command += ["stylelint", f"ST_ARGS={file_paths}"]
+        run_command += ["stylelint", f"ST_ARGS=-- --fix {file_paths}"]
     elif target == "misc":
         run_command += ["check-package-lock"]
 
     returncode: int = subprocess.call(run_command)
+    subprocess.call(["git", "add"] + args.file_paths)
```

Comment:
`args.file_paths` is `Any` type, so could you add a type like the following

```diff
diff --git a/pre-commit/web_lint_check.py b/pre-commit/web_lint_check.py
index 5c8197b2f..a3c36df7f 100755
--- a/pre-commit/web_lint_check.py
+++ b/pre-commit/web_lint_check.py
@@ -4,6 +4,7 @@ import argparse
 import multiprocessing
 import os
 import subprocess
+from typing import List
 
 
 def web_lint_check():
@@ -14,7 +15,8 @@ def web_lint_check():
     DIR = "webui/react/"
 
     target: str = args.target
-    file_paths: str = " ".join([os.path.relpath(file_path, DIR) for file_path in args.file_paths])
+    args_file_paths: List[str] = args.file_paths
+    file_paths: str = " ".join([os.path.relpath(file_path, DIR) for file_path in args_file_paths])
     nproc: int = multiprocessing.cpu_count()
     run_command: list[str] = [
         "make",
@@ -34,7 +36,7 @@ def web_lint_check():
         run_command += ["check-package-lock"]
 
     returncode: int = subprocess.call(run_command)
-    subprocess.call(["git", "add"] + args.file_paths)
+    subprocess.call(["git", "add"] + args_file_paths)
     exit(returncode)
```

Reply:
actually, we can do `parser.add_argument("file_paths", help="1 or more file paths", nargs="+", type=List[str])` too, but type wasnt correctly assigned on my local

Reply:
the second method doesn't seem to communicate the type to the type checker, does it?